includes/data-stores/abstract-wc-order-data-store-cpt.php

```diff
@@ -223,9 +235,15 @@ protected function update_post_meta( &$order ) {
 
 		$props_to_update = $this->get_props_to_update( $order, $meta_key_to_props );
 		foreach ( $props_to_update as $meta_key => $prop ) {
-			$value   = $order->{"get_$prop"}( 'edit' );
-			$updated = update_post_meta( $order->get_id(), $meta_key, $value );
-			if ( $updated ) {
+			$value = $order->{"get_$prop"}( 'edit' );
+
+			switch ( $prop ) {
```

Comment:
The `switch` with only one case is here to make it easy to add more things to this logic in the future, correct? Otherwise it seems like an `if` would be simpler.